app/src/main/java/com/summer/practice/tvtracker/Registration.kt

```diff
@@ -0,0 +1,12 @@
+package com.summer.practice.tvtracker
```

Comment:
let's also create a new package called register and place this activity inside of it